### ISSUE TITLE
Consider making available as CocoaPod

### ISSUE THREAD

**Issue by samdods** [theappbusiness/ConfigGenerator#16]
as requested by @jtbthethird in issue #1

**Comment by ealeksandrov** [comment]
I don't think that such tool should be connected as cocoapod.
It will require different settings for different targets, I'm not sure it is possible at all. And different developers can prefer completely different approach to splitting project into environments (see https://github.com/theappbusiness/ConfigGenerator/issues/21 for example).

**Comment by KaneCheshire** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Issue #16: Consider making available as CocoaPod](https://trello.com/c/XmUdPpm5/16-issue-16-consider-making-available-as-cocoapod)

**Comment by phatblat** [comment]
I agree with @ealeksandrov that this tool doesn't make sense as a CocoaPod. While a pod can be installed in [some or all build configurations](https://guides.cocoapods.org/syntax/podfile.html#pod), it is difficult to do anything fancy with the limited [pre/post-install hooks](https://guides.cocoapods.org/syntax/podfile.html#post_install). 

The only thing that might be useful is the relatively new pod [script_phase](https://guides.cocoapods.org/syntax/podfile.html#script_phase), which allows a pod to install a run script into the build phases of the configured target.
